src/app/components/ListSelector/ListSelector.jsx

```diff
@@ -36,7 +36,7 @@ class ListSelector extends React.Component {
     currentSeason = '',
     listType = 'staff-picks',
     filters = [],
-    selectedFilters = []
+    selectedFilters = [],
```

Comment:
This should not have the trailing `,` .It is one of the arguments of a function.